### ISSUE TITLE
DNS hosting 

### ISSUE THREAD

**Issue by wlonkly** [irchelp/wio#172]
Hey folks,

I've been hosting DNS for irchelp.org for a while, and it's at my Linode account which I don't use for anything else anymore. I'd like to either:

1. hand that over to someone that's still active with wio/#irchelp, or failing that
2. migrate it to my DigitalOcean account which I at least use for other things.

Let me know if (1) is a possibility, and failing that let me know and I'll arrange for (2). I don't have access to GoDaddy to make the change myself.

Thanks!
Rich/mendel

**Comment by a-raccoon** [comment]
asking around.  thanks.

**Comment by toocool60608** [comment]
Is this the list of contacts for the website at the bottom of the page
https://www.godaddy.com/whois/results.aspx?checkAvail=1&domain=irchelp.org
Not sure which is the best person / position to contact

**Comment by a-raccoon** [comment]
i already elected you, toocool. stop shirking your duties! :)

**Comment by wlonkly** [comment]
I have a _feeling_ jolo holds the keys, if that helps. I think I tried to email him at the last contact info I had a couple years ago when I moved my own hosting off of Linode, to no avail.

**Comment by sdaugherty** [comment]
Could be davin too

**Comment by wlonkly** [comment]
Checking in -- any news?

**Comment by a-raccoon** [comment]
I tried Jolo's phone number, and a different email address referenced in his voicemail.  I don't know how to contact davin.

**Comment by wlonkly** [comment]
Huh, rats. I'm glad we're figuring this out when stuff is working, and not when it's urgent!

Checked my address book -- I have `joseph dot lo at duke dot edu` for him but I also see that I emailed him at that address back in 2017 (for this reason!) to no response. I don't have any contact info for Davin though.

I was going to also try the old mailing list, but I see the MX for irchelp.org is not accepting connections on port 25, so so much for THAT plan.

**Comment by a-raccoon** [comment]
His voicemail references his 4 letter nick at duke edu, but I estimate he should receive email at both given he had the other one for so long.  I should try actually leaving him a voicemail message as well just in case.

**Comment by jolo123** [comment]
Hello world! Just seeing all these messages. Sorry I am in email bankruptcy.

For GoDaddy, I am the only active person left, as others like Davin have fallen off. Alex is still listed as technical contact, but I'm pretty sure that's a dead account. I am the registrant, admin, and billing contact. We should at least make technical contact 1 of you, in case I get hit by a bus. 

I will defer to you guys on how best to handle the DNS. We can certainly move DNS to GoDaddy or DigitalOcean. Let me know what you decide?

**Comment by jolo123** [comment]
Semi-related, but WIO itself has also been on autopilot since ~2013. Last I remember, it was hosted by Doug Boldt and curated by Stephanie, who was trying to update something but she had some weird SSL cert issue and could never get through. I tried sending an email to irchelplist last year, but it bounced because that has been dead since Alex left. What's the easiest way to manage this? Do the current ops have a new list?

I would love to hand over the keys to one of you, or someone else trusted and active. At this point, some curation is better than none, and I don't think we should even try to be perfect. :)

**Comment by jolo123** [comment]
I can't even remember why we didn't just use GoDaddy, since we're paying for it? (This was all before GoDaddy became a somewhat yucky service but that's a separate issue.) Might just be legacy, ie, we've got it working fine at [the thing before linode], why change?

If there are no objections, then I can switch to GoDaddy with a single click. (I just checked.)

**Comment by wlonkly** [comment]
Yeah I think that's it, just only changed the part we needed to. Way back, it was hosted on Chris Petro's (wankel) coloed server (mankind.boredom.org) via me, and then I moved most of "my stuff" from there to Linode because the colo deal went away, and then at some point wio moved to Github Pages but there was no need to change nameservers, and so on.

Here's the zone:

```
irchelp.org.		86400	IN	SOA	ns1.linode.com. hostmaster.mati.ca. 2016071455 14400 14400 1209600 86400
irchelp.org.		86400	IN	NS	ns1.linode.com.
irchelp.org.		86400	IN	NS	ns2.linode.com.
irchelp.org.		86400	IN	NS	ns3.linode.com.
irchelp.org.		86400	IN	NS	ns4.linode.com.
irchelp.org.		86400	IN	NS	ns5.linode.com.
irchelp.org.		86400	IN	MX	10 alpha.spacehost.us.
irchelp.org.		86400	IN	A	192.30.252.153
irchelp.org.		86400	IN	A	192.30.252.154
ircnet.irchelp.org.	86400	IN	A	213.95.21.31
preview.irchelp.org.	86400	IN	CNAME	irchelp.github.io.
vanity.irchelp.org.	86400	IN	CNAME	deckard.duhs.duke.edu.
www.irchelp.org.	86400	IN	CNAME	irchelp.github.io.
www2.irchelp.org.	86400	IN	A	216.246.34.83
```

**Comment by a-raccoon** [comment]
I don't know enough about the deep history of GoDaddy and their DNS capabilities and limitations to say no.  It was fine when I used GoDaddy 15-20 years ago, and I've been using NameCheap's own DNS (the registrar I'd been using ever since).  But I've never done anything fine-tuned with DNS to know any better.

As far as site maintenance, I was going to start another Issue to ask Stephanie her thoughts on us eliminating the preview.irchelp.org and built straight to www.irchelp.org from github.  If you're not aware, Jolo, the site is being hosted by Github Pages and everything can be done through here via permissions granted on Github.

I just don't fully understand the multi-faceted build process myself to be very useful.  I do wish it were a bit more simplified so I can push updates from the numerous contributions and pull requests made over the past several years. See our --> [Pull Requests](https://github.com/irchelp/wio/pulls?q=).

I know that TooCool and lil0ne are rather active and do their own network/web stuff and IRC admin stuff so they're competent, capable and engaged.  EmleyMoor is too, but he doesn't talk as much these days so I don't know how he feels about being roped in.  If you gave me the access codes, they'll sit in my little black book until the next emergency arises and I can beg someone into fixing it, until I get hit by a bus.

**Comment by wlonkly** [comment]
Oh hey I just noticed this bit:

> We should at least make technical contact 1 of you, in case I get hit by a bus.

I don't think technical contact would get you any sway with GoDaddy, the point of those separate contacts is pretty outdated at this point. Better that people have keys to the GoDaddy account itself to reduce the, uh, lottery number. :)

**Comment by jolo123** [comment]
So to recap, we will switch DNS to GoDaddy, and add Raccoon as tech contact (even though it might not help much, but can't hurt to at least have 1 other person to answer the phone). 

But the biggest change is for me to add some/all of you guys to have GoDaddy account access. It looks like they actually have this option, so that others can help manage the domain and "products" without being able to mess with my payment system:
https://www.godaddy.com/help/delegate-access-levels-of-permission-12374

I could add Rich, just so he can manage the transition. Switching to GoDaddy DNS is just throwing a switch, but there are no other options shown, at least not on that first screen. So I don't know how to map all the entries over (A, MX, C, etc.). I did this years ago for my other domain, just can't remember details. Perhaps after the changeover, there will be more options shown?

I can then also add Raccoon and TooCool. Don't think there is a limit. Unless I hear otherwise, I'll use the email addresses used to contact me recently.

**Comment by wlonkly** [comment]
Sounds good to me -- I'm happy to do the switchover, and then I'll have keys as a last resort if nobody that's still active is reachable.

**Comment by a-raccoon** [comment]
I don't know if I really want my personal details in the whois record associated with this domain.  way to dox myself on efnet! :o

**Comment by jolo123** [comment]
Let me start by inviting Rich to have the delegated authority. We'll see how that goes. I think his email won't get spammed onto the whois record, it's just an internal thing, kind of like me sharing my login credentials but in a more compartmentalized way.

**Comment by wlonkly** [comment]
I have access! I'll move stuff over in the next day or two and update here when I've done so.

**Comment by jolo123** [comment]
Yay! And of course now I see where one might make the changes, but I'm still glad for you to do it, so I don't scramble things up. Thanks again to all.

We can take the WIO issue to a separate thread. It would definitely be good to come up with a sustainable option.

**Comment by jolo123** [comment]
This was 1st time I've seen github used this way, very interesting. (I am too old to code any more, so I don't use git.) Lemme know when it's time to jump over to WIO issue, or if you want to just get Stephanie's attention and we just keep going here. I presume this is semi private?

**Comment by a-raccoon** [comment]
This is super public.  We should avoid/redact private things put here, which is also why care and attention was given to what files migrated from the old site to the github rendition.

I'll start a new Issue now.

**Comment by sdaugherty** [comment]
I'm following the issue, and we can start a new one to figure out what's
next - or I guess we could actually get on IRC for the first time in
forever.

**Comment by wlonkly** [comment]
And done! I copied the zone as-is, which means that the MX record is still in place, etc. I'll leave the zone at Linode for a week for any cached stragglers. Let me know if anything seems amiss.

**Comment by jolo123** [comment]
Per Raccoon's suggestion, also added TooCool to the GoDaddy delegation list.
